lib/src/theme/colors.dart

```diff
@@ -106,6 +123,14 @@ class DSFRColors extends ThemeExtension<DSFRColors> {
           badgeWarning: ColorPalette.warning950,
           badgeNews: ColorPalette.yellowTournesol950,
           banner: ColorPalette.info950,
+          radioActive: ColorPalette.blueFranceSun113,
+          frLabel: const Color(0xFF1e1e1e),
+          frFieldsetDisabled: const Color(0xFF6a6a6a),
+          frHintText: const Color(0xFF6a6a6a),
+          g200: const Color(0xFFf0f0f0),
```

Comment:
why this doesn't use the ColorPalette class?

Reply:
I actually prefer to define colors directly inside the colors.dart file, at first the color palette was made only to have some base color as described in the design system but there not all named, for example there is no specific name for the `0xFFf0f0f0` in the design system.

Reply:
Yeah I have saw this kind of mistakes in the DSFR design system. I think that if the color is define here https://gouvfr.atlassian.net/wiki/spaces/DB/pages/217186370/Couleurs+-+palette+-+Color we should put it in palette.dart.

And that's the case for:

- 1e1e1e -> grey75

this color: 6a6a6a is used two times, so maybe we can put it in a variable

Reply:
Seems ok to me I'll make the modification.